modules/scheduler/gke-cluster/main.tf

```diff
@@ -209,6 +209,15 @@ resource "google_container_cluster" "gke_cluster" {
       condition     = !(!coalesce(var.enable_multi_networking, true) && length(var.additional_networks) > 0)
       error_message = "'enable_multi_networking' cannot be false when using multivpc module, which passes additional_networks."
     }
+    precondition {
```

Comment:
nit: these would be better as simple validation blocks on the variables themselves. I believe this will also fail pretty early, but a validation block in variable is guaranteed to fail at "terraform plan" or "terraform apply" before anything is created. For example:

https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/1a1e22a252c8b5bf0be57fa44f46bf72ca0363bf/modules/compute/vm-instance/variables.tf#L230-L235

Reply:
Ack, done